src/platforms/flutter/troubleshooting.mdx

```diff
@@ -13,6 +13,7 @@ If you need help solving issues with Sentry's Flutter SDK, you can read the edge
 - If you enable the `split-debug-info` and `obfuscate` features, you must upload [debug symbols](/platforms/flutter/upload-debug/).
 - Issue titles might be obfuscated as we rely on the `runtimeType`, but they may not be human-readable. See the [Obfuscate Caveat](https://flutter.dev/docs/deployment/obfuscate#caveat).
 - Layout related errors are only caught by [FlutterError.onError](https://api.flutter.dev/flutter/foundation/FlutterError/onError.html) in debug mode. In release mode, they are removed by the Flutter framework. See [Flutter build modes](https://flutter.dev/docs/testing/build-modes).
+- Use [inbound filters](https://docs.sentry.io/product/data-management-settings/filtering/) to exclude unhandled errors that are caught outside of your application in release builds. The SDK cannot filter these directly due to obfuscated stack traces.
```

Comment:
If you include the https://docs.sentry.io/ prefix the link is formatted as if it's external:
![Screenshot 2023-12-04 at 1 27 16 PM](https://github.com/getsentry/sentry-docs/assets/3520671/abcbf862-0f19-4e41-a625-abed9272ea57)

```suggestion
- Use [inbound filters](/product/data-management-settings/filtering/) to exclude unhandled errors that are caught outside of your application in release builds. The SDK cannot filter these directly due to obfuscated stack traces.
```